factory-reset-lvm

```diff
@@ -147,33 +159,27 @@ fi
 # Create home LV
 lvm lvcreate -L "$HOME_SIZE"K --name home sailfish
 
-# Create filesystems
-mkfs.ext4 $ROOTDEV
-mkfs.ext4 $HOMEDEV
-
 # Start restoring Sailfish OS from the factory images
-mount $ROOTDEV $MOUNT_POINT
-mkdir -p $MOUNT_POINT/home
-mount $HOMEDEV $MOUNT_POINT/home
-TEMPMOUNT=$(mktemp -d)
 
-mount -t squashfs -o loop $SAILFISH_FIMAGE/sailfish_root.squashfs.img $TEMPMOUNT
-cp -a $TEMPMOUNT/* $MOUNT_POINT
+lzopcat $SAILFISH_FIMAGE/root.img.lzo > $ROOTDEV
 sync
```

Comment:
we don't nee this sync here do we? One sync is enough after all partitions are done right?